README.md

```diff
@@ -174,6 +174,7 @@ Please refer to [installation.md](docs/en/installation.md) for installation.
   ```bash
   > mim download mmcls --config resnet18_8xb16_cifar10
   > mim download mmcls --config resnet18_8xb16_cifar10 --dest .
+  > mim download mmcls --no-check-certificate --config resnet18_8xb16_cifar10 --dest .
```

Comment:
Since the `--no-check-certificate` is not a commonly used option. I don't think it's necessary to update the README. ^_^

Reply:
@ice-tong Thank you for your review!
Then, how to edit the code? I am contributed to github for the first time.